syncd/syncd.cpp

```diff
@@ -555,6 +560,11 @@ int main(int argc, char **argv)
     getRequest = new swss::ConsumerTable(db, "GETREQUEST");
     getResponse  = new swss::ProducerTable(db, "GETRESPONSE");
 
+#ifdef MLNXSAI
+    std::string mlnx_config_file = "/etc/ssw/ACS-MSN2700/sai_2700.xml";
```

Comment:
currently this can be hardcoded but at later stage this could be passed from cmdline